### PR TITLE
Added Tag icon on hover, extended popover with 'Original name:' prefix

### PR DESCRIPTION
**Pull request by stankis**

### CODE REVIEW

File: src/ui/components/Navigation/Core/TableView/TableView.scss [hunk 1]
```diff
@@ -249,6 +249,33 @@
...
+        margin-right: 5px;
+        min-width: 28px;
+        min-height: 28px;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        cursor: pointer;
+        border-radius: 4px;
+
+        &_isVisible {
```
Comment (imsitnikov) [inline src/ui/components/Navigation/Core/TableView/TableView.scss:263]: `isVisible` -> `visible`
let's not use camelCase in css classes

File: src/ui/components/Navigation/Core/FavoritesNameWithAliasItem/FavoritesNameWithAliasItem.tsx [hunk 2]
```diff
@@ -41,16 +44,23 @@ export const FavoritesNameWithAliasItem = (props: FavoritesNameWithAliasItemProp
...
-                    <div
-                        className={b('row-btn', b('edit-alias-btn', {visible: isAliasVisible}))}
-                        onClick={onLabelClick}
+
+            <div className={className} onClick={onLabelClick}>
+                {isAliasVisible ? (
+                    <Popover
+                        placement={['right', 'left', 'bottom', 'top']}
+                        content={
+                            <>
```
Comment (imsitnikov) [inline src/ui/components/Navigation/Core/FavoritesNameWithAliasItem/FavoritesNameWithAliasItem.tsx:53]: `<>` -> `<React.Fragment>`

File: src/ui/components/Navigation/Core/TableView/TableView.scss [hunk 3]
```diff
@@ -249,6 +249,33 @@
         }
     }
 
+    &__edit-favorites-alias-btn {
+        visibility: hidden;
+        margin-right: 5px;
```
Comment (imsitnikov) [inline src/ui/components/Navigation/Core/TableView/TableView.scss:254]: Can we use here only `visibility` prop and move the rest of the props to `FavoritesNameWithAliasItem.scss`?
Looks like styling a child component from its parent. Not critical, final decision up to you.

**Review by imsitnikov — CHANGES_REQUESTED** [review]

File: src/ui/components/Navigation/Core/FavoritesNameWithAliasItem/FavoritesNameWithAliasItem.tsx (hunk 2 above)
Comment (stankis) [reply src/ui/components/Navigation/Core/FavoritesNameWithAliasItem/FavoritesNameWithAliasItem.tsx:53]: done, but why ? <> looks shorter and no need for setting attributes

File: src/ui/components/Navigation/Core/TableView/TableView.scss (hunk 3 above)
Comment (stankis) [reply src/ui/components/Navigation/Core/TableView/TableView.scss:254]: Yes, agree, refactored

File: src/ui/components/Navigation/Core/TableView/TableView.scss (hunk 1 above)
Comment (stankis) [reply src/ui/components/Navigation/Core/TableView/TableView.scss:263]: no problem, fixed

**Review by imsitnikov — APPROVED** [review]

**Pull request merged**